packages/api-client/src/api/getProducts/index.ts

```diff
@@ -14,8 +14,13 @@ export default async function getProducts({ client, config }: ApiContext, params
       include = 'default_variant,variants.option_values,option_types,taxons,images';
     }
 
+    const unique = Array.from(new Set(productPropertyFilters.map(filter => filter.productPropertyName)).values())
+    let properties = {};
+    for (var uniqueProperty of unique){
+       properties[uniqueProperty] = (productPropertyFilters.filter(property => property.productPropertyName === uniqueProperty).map(property => property.productPropertyValue)).join(',')
```

Comment:
How about extracting the `productPropertyFilters.filter(property => property.productPropertyName === uniqueProperty` to a separate variable?
Then the map portion to another one, and finally assign it to properties[uniqueProperty]? This way it will be easier to read.